client/package.json

```diff
@@ -19,6 +19,7 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
+  "proxy": "http://localhost:8080",
```

Comment:
Why is this necessary? 🤔 we can pair on getting the host as an environment variable or something in the future